src/sql/workbench/services/notebook/browser/models/notebookUtils.ts

```diff
@@ -72,3 +73,19 @@ export async function asyncForEach(array: any[], callback: Function): Promise<an
 		}
 	}
 }
+
+/**
+ * Check if a given file path is valid
+ * @param href the filepath as string
+ * @returns true if a file path is valid
+ */
+export function isFilepathValid(href: string): boolean {
+	fs.stat(href, (error, stats) => {
```

Comment:
This function is always returning true. You need to either create a promise and then call resolve with true/false in the callback or (preferably) use [pfs.Promises.stat](https://github.com/Microsoft/azuredatastudio/blob/main/src/vs/base/node/pfs.ts#L257)

(you should not be using fs directly in core code. That's why you're getting the hygiene failure in the build)